lib/cog/models/command.ex

```diff
@@ -50,14 +50,53 @@ defmodule Cog.Models.Command do
     end
   end
 
+  def parse_name(name) do
+    case parse_qualified_name(name) do
+      {:ok, command} ->
+        {:ok, command}
+      {:error, _} ->
+        parse_ambiguous_name(name)
+    end
+  end
+
+  def parse_qualified_name(name) do
+    case String.split(name, ":", parts: 2) do
```

Comment:
It's actually right above this, but has some weird functionality. I think we still use it in a few places so I didn't rip it out. Let me see if I can without too much trouble.